audio/midiplayer.h

```diff
@@ -105,7 +105,7 @@ class MidiPlayer : public MidiDriver_BASE {
 	 *
 	 * @todo This method currently does not do anything if the new volume
 	 *       matches the old volume. But some engines always update the
-	 *       volume (Parallaction, Tinsel, Touche, ...). This is why
+	 *       volume (Parallaction, Tinsel, Touché, ...). This is why
```

Comment:
This references the engine, not the game title

Reply:
Changed to `Touche`, which is consistent with others.